snekbox.py

```diff
@@ -27,13 +29,23 @@ def __init__(self,
         'PYTHONDONTWRITEBYTECODE': '1',
     }
 
-    def nsjail_workaround(self):
+    def _nsjail_workaround(self):
```

Comment:
Missing docstring. To be fair I have no idea what this workaround is.

Reply:
I made it a private method, which don't require docstrings

Reply:
I didn't know what it did either, and it was only used internally, hence making it private